src/pages/Home.vue

```diff
@@ -67,10 +81,17 @@ export default {
       default: "ID",
       type: String,
     },
+    /** ID of session */
+    sessionId: {
+      default: "",
```

Comment:
How about making the default as `null`?
And then checking in the code if `sessionId == null`?